catalog/tests/dags/providers/provider_api_scripts/test_brooklyn_museum.py

```diff
@@ -1,20 +1,26 @@
+import json
+from pathlib import Path
 from unittest.mock import patch
 
 import pytest
 
-from catalog.tests.dags.providers.provider_api_scripts.resources.json_load import (
-    make_resource_json_func,
-)
-from common.licenses import LicenseInfo
+from common.licenses import get_license_info
 from common.loader import provider_details as prov
 from common.storage.image import ImageStore
 from providers.provider_api_scripts.brooklyn_museum import BrooklynMuseumDataIngester
 
 
+RESOURCES = Path(__file__).parent / "resources/brooklynmuseum"
 bkm = BrooklynMuseumDataIngester()
 image_store = ImageStore(provider=prov.BROOKLYN_DEFAULT_PROVIDER)
 bkm.media_stores = {"image": image_store}
-_get_resource_json = make_resource_json_func("brooklynmuseum")
+
+CC_BY_3_0 = get_license_info("https://creativecommons.org/licenses/by/3.0/")
+CC0 = get_license_info("https://creativecommons.org/publicdomain/zero/1.0/")
+
+
+def _get_resource_json(json_name):
+    return json.loads((RESOURCES / json_name).read_text())
```

Comment:
Why was this change reverted? These function were unified in https://github.com/WordPress/openverse-catalog/pull/1059

Reply:
I have no idea how this happened 🤯  Thank you for catching!